### PR TITLE
[WIP] Build page data

### PR DESCRIPTION
**Pull request by flynnduism**
* configure buildComponent to use buildService
* replace static content with data from API
* configure router for a static /build view
* tidy up any layout and/or styling issues

### CODE REVIEW

File: src/app/in-memory-data.service.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { InMemoryDbService } from 'angular-in-memory-web-api';
 import { projectBuilds } from './mock/project-builds';
 import { projects } from './mock/projects';
+import { builds } from './mock/builds';
 
 export class InMemoryDataService implements InMemoryDbService {
   createDb() {
-    return { projectBuilds, projects };
+    return { projectBuilds, projects, builds };
```
Comment (seanknox) [inline src/app/in-memory-data.service.ts]: we ditched the in memory data service for a different mock data service Radu put together. you'll want to rebase sooner or later before it gets too painful.

File: src/app/mock/builds.ts [hunk 2]
```diff
@@ -0,0 +1,257 @@
+export const builds = [
```
Comment (seanknox) [inline src/app/mock/builds.ts]: Ditch this for whatever is in `angular5`.

**Comment by seanknox** [comment]
@flynnduism due to the many changes in mock data I applied your changes onto a new branch at #167. Would recommend we close this one and finish over in #167.

**Comment by flynnduism** [comment]
Done and done. Thanks @seanknox

**Pull request closed without merging**